package.json

```diff
@@ -35,18 +32,19 @@
     "url": "git+https://github.com/jdiaz5513/capnp-ts.git"
   },
   "scripts": {
-    "bootstrap": "lerna bootstrap",
+    "bootstrap": "lerna bootstrap --hoist",
     "benchmark": "gulp benchmark",
     "build": "gulp build",
-    "ci": "gulp lint && gulp coverage && codecov --disable=gcov",
+    "ci": "gulp lint && gulp coverage && codecov --disable=gcov && cd js-examples && ./test.sh",
```

Comment:
The upstream source does it. We want simplicity in this package as it serves as the basic usage example. But I'm happy to add a .bat for Windows perhaps?